src/coriolis.jl

```diff
@@ -33,12 +33,13 @@ If `f` is not specified, it is calculated from `rotation_rate` and
 Also called `FPlane`, after the "f-plane" approximation for the local effect of
 Earth's rotation in a planar coordinate system tangent to the Earth's surface.
 """
-function FPlane(FT=Float64; f=nothing, rotation_rate=nothing, latitude=nothing)
+function FPlane(; f=nothing, rotation_rate=nothing, latitude=nothing)
```

Comment:
Is this the best solution, or should we dispatch on the type of `FT`?

Reply:
Hmmm, dispatching on the type of `FT` could work but then you'd have to write a new function for each `FT` (might be fine until we start playing around with posits) but would also be breaking because an `FT` must be passed.

Right now `FPlane(f=1e-4)` is still valid but if we dispatched on `FT` then you'd always have to supply an `FT` to dispatch on, e.g. `FPlane(Float64; f=1e-4)` which I feel breaks more code as we usually run with the default `Float64`.

Reply:
The type of `FT` is always `DataType`, is it not?

Reply:
The problem is that this method conflicts with the default constructor `FPlane(f)`, where `f` is expected to be a `Number` of some kind, probably.

Reply:
Ah of course, that's the most sensible solution. I feel pretty dumb now.

Just committed this change.

Reply:
Well, I got stung by the same problem some time earlier (not sure why the dispatch was removed...), which is why I knew how to fix it.